misc-tools/dj_make_chroot.in

```diff
@@ -156,7 +156,7 @@ if [ "$DISTRO" = 'Debian' ]; then
     INCLUDEDEBS="ca-certificates"
 
     # Packages to install after upgrade (space separated):
-    INSTALLDEBS="gcc g++ make default-jdk-headless default-jre-headless pypy3 python3 locales"
```

Comment:
How is this related to Python 2?

Reply:
This is not related to it, but the removal in L188 is.

Reply:
So this was more part of the commit message: `Default we set compiling via pypy3 so the python3 is not needed anymore.`

If you prefer I can split this in 2 messages?

Reply:
Yes, please add a separate commit/message and would also be good to refer to https://github.com/DOMjudge/domjudge/pull/914 in it.